libs/utils/utils.js

```diff
@@ -306,6 +348,38 @@ export const getFederatedUrl = (url = '') => {
   return url;
 };
 
+export function hasLanguageLinks(area) {
+  for (const link of area.querySelectorAll('a[href]')) {
```

Comment:
Why not use something like ``area.querySelectorAll(`a[href*="${your-domain}"]`)``

Reply:
This was a great idea. Unfortuntately I won't be able to use it as I realized that I had to cover some more cases for these links (adobe.com/de/express for example) and the logic has now changed to accomodate that.